docs/user/guides/subprojects.rst

```diff
@@ -0,0 +1,20 @@
+How To Add a Subproject
+=======================
+
+This guide shows you how to add a Subproject to your main project.
+
+.. seealso::
+
+   :doc:`/subprojects`
+      Read more about what the subproject feature can do and how it works.
+
+Adding a subproject
+-------------------
+
+In the admin dashboard for your project, select "Subprojects" from the menu.
+From this page you can add a subproject by typing in the project slug.
```

Comment:
```suggestion
From this page you can add a subproject by typing in the project slug.

.. note::

   * On |org_brand|, you need to be maintainer or owner of a subproject in order to choose it from your main project.
   * On |com_brand|, you need to be Owner or have write access to the subproject in order to choose it from your main project.
```

Reply:
Is this accurate? CC: @humitos @ericholscher ?

Reply:
Write access isn't really a concept on .com. I think you mean admin permissions?

Reply:
Updated the suggestion and added it :+1: